utils.py

```diff
@@ -137,27 +137,17 @@ def multipart_encode(raw_crash, boundary=None):
     return output, headers
 
 
-def post_crash(url, payload, headers, size, compressed=False):
+def post_crash(url, payload, headers):
```

Comment:
I took out the compressed bits here since we do that when generating the crash payload and don't need to do it here.